### PR TITLE
Posit.Quarto version 1.3.340

### PR DESCRIPTION
**Pull request by dpprdan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103701)

### CODE REVIEW

**Comment by dpprdan** [comment]
Compared to the https://github.com/microsoft/winget-pkgs/tree/master/manifests/p/Posit/Quarto/1.2.475 manifest, this does not include `AppsAndFeaturesEntries.UpgradeCode` (should this still be necessary, where would I get it with wingetcreate or SandboxCreate?) 
`ProductCode` is now under `Installers` (as provided by wingetcreate).

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
> Compared to the https://github.com/microsoft/winget-pkgs/tree/master/manifests/p/Posit/Quarto/1.2.475 manifest, this does not include `AppsAndFeaturesEntries.UpgradeCode` (should this still be necessary, where would I get it with wingetcreate or SandboxCreate?) `ProductCode` is now under `Installers` (as provided by wingetcreate).

Since it passed validation, it might not be necessary.

**Comment by cderv** [comment]
Thanks @dpprdan !

File: manifests/p/Posit/Quarto/1.3.340/Posit.Quarto.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Posit.Quarto
+PackageVersion: 1.3.340
+PackageLocale: zh-CN
+Publisher: Rstudio, PBC
```
Comment (cderv) [inline manifests/p/Posit/Quarto/1.3.340/Posit.Quarto.locale.zh-CN.yaml:7]: @dpprdan Just wondering why not change the Publisher ultimately ? 
I think you did that for the other tool right ? https://github.com/microsoft/winget-pkgs/pull/100319

Comment (dpprdan) [reply manifests/p/Posit/Quarto/1.3.340/Posit.Quarto.locale.zh-CN.yaml:7]: @cderv You're right, Publisher should be Posit now. I see you opened https://github.com/microsoft/winget-pkgs/pull/105250. 👍🏻

Comment (dpprdan) [reply manifests/p/Posit/Quarto/1.3.340/Posit.Quarto.locale.zh-CN.yaml:7]: @cderv Now I remember why I didn't change the Publisher to Posit: 
"Rstudio, PBC" (sic!) is still showing up as the Publisher in the ARP dialog. (For Posit.RStudio it is indeed Posit Software)

grafik

[SandboxTest.ps1](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#how-do-i-get-the-appsandfeaturesentries-for-an-installer) reports accordingly:

```
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.Admin setting enabled.

--> Installing the Manifest 1.3.340

Found Quarto [Posit.Quarto] Version 1.3.340
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/quarto-dev/quarto-cli/releases/download/v1.3.340/quarto-1.3.340-win.msi
  ██████████████████████████████  77.8 MB / 77.8 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName DisplayVersion Publisher    ProductCode
----------- -------------- ---------    -----------
Quarto      1.3.340        Rstudio, PBC {84C4DFFB-EAE4-4023-BF77-E56014E4D240}

```

My guess would be that it needs to be changed here:
https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/windows/quarto.wxs#L4

Vaguely related, I also found [this](https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/linux/installer.ts#L82) and [that](https://github.com/quarto-dev/quarto-cli/blob/4a4e9f1226df1f6f994ac084b226150966bc7aa5/package/src/linux/installer.ts#L108).

**Pull request merged**